<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitDialog</class>
 <widget class="QDialog" name="GitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>295</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Git</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
background:rgb(233,237,245);
}

QLineEdit
{
border-style:none;
padding:3px;
border-radius:7px;
background:#dde3f0;
}

QPlainTextEdit
{
border-style:none;
padding:3px;
border-radius:7px;
background:#dde3f0;
}

QPushButton
{
border-style:none;
padding:3px;
border-radius:7px;
border:1px solid rgb(119,119,119);
background:rgb(220,227,242);
}

QPushButton:hover
{
background:rgb(250, 250, 250);
}

QPushButton:pressed
{
background:rgb(220, 220, 220);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt; Git路径:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;文件路径:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;提交注释:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="NotesEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="GitPathBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FilePathBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CommitBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border-style:none;
padding:3px;
border-radius:7px;
background:#dde3f0;
}

QPushButton:hover
{
background:rgb(250, 250, 250);
}

QPushButton:pressed
{
background:rgb(220, 220, 220);
}</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="GitPathEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="FilePathEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Git&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Notes">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-style:italic; text-decoration: underline;&quot;&gt;注：使用前请配置并创建Git本地仓库&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>GitDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
